d3a_api_client/setups/test_market_aggregator.py

```diff
@@ -58,22 +66,16 @@ def on_finish(self, finish_info):
     "domain_name": domain_name,
     "websockets_domain_name": websockets_domain_name
 }
-
+market_names = ["house 1"]
 house_1_uuid = get_area_uuid_from_area_name_and_collaboration_id(
-    market_args["simulation_id"], "House 1", market_args["domain_name"])
+    market_args["simulation_id"], market_names[0], market_args["domain_name"])
```

Comment:
Is this change for later, when we want to connect to more than one market?

Reply:
Yes, If we add more houses then we have to add the house name to the list and access its index value again.